modules/loader/src/main/java/com/opengamma/strata/loader/csv/FullSwapTradeCsvPlugin.java

```diff
@@ -266,6 +277,39 @@ private static RateCalculationSwapLeg parseLeg(
     PeriodicSchedule accrualSch = parseAccrualSchedule(row, leg);
     PaymentSchedule paymentSch = parsePaymentSchedule(row, leg, accrualSch.getFrequency());
     NotionalSchedule notionalSch = parseNotionalSchedule(row, leg);
+
+    Optional<String> knownAmount = findValue(row, leg, KNOWN_AMOUNT_FIELD);
+    if (knownAmount.isPresent()) {
+      return KnownAmountSwapLeg.builder()
+          .payReceive(payReceive)
+          .accrualSchedule(accrualSch)
+          .paymentSchedule(paymentSch)
+          .currency(notionalSch.getCurrency())
+          .amount(ValueSchedule.of(LoaderUtils.parseDouble(knownAmount.get())))
+          .build();
+    }
+
+    return parseRateCalculationLeg(
+        row,
+        leg,
+        index,
+        defaultFixedLegDayCount,
+        payReceive,
+        accrualSch,
+        paymentSch,
+        notionalSch);
+  }
+
+  private static SwapLeg parseRateCalculationLeg(
```

Comment:
Note that all the other private methods have a `//` comment, so this one should too.